package.json

```diff
@@ -12,6 +12,9 @@
     "test-ts": "hardhat --network hardhat test"
   },
   "devDependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "1.0.4",
```

Comment:
it's very cool that you included an exact-version dependency

try however to remember to update the lockfile in the same commit as when you add a dependency